README.md

```diff
@@ -20,7 +20,7 @@ the "what" and "why" of data breach alerts.
 
 ### Requirements
 
-* [Node](https://nodejs.org/) (with NPM): we recommend using [Node Version Manager](https://github.com/nvm-sh/nvm) to install and manage Node/NPM.
+* [Volta](https://volta.sh/) (installs the correct version of Node and npm)
```

Comment:
suggestion: should this be added under a new "Recommended tools" section?  Something like Postgres is required, but Volta is not.  Also, I have some other recommendations to add, should such a section exist 😉

Reply:
Yeah, that sounds like a good idea - Volta might not be required, but using the same version of Node and particularly npm is. Would you mind submitting a separate PR with those recommended tools? Then we can iterate on that list in that PR, and just replace this with something like `the correct version of Node and npm - we [recommend](#recommended-tools) installing Volta to take care of this`?